local/network.go

```diff
@@ -593,16 +604,23 @@ func (ln *localNetwork) removeNode(nodeName string) error {
 	delete(ln.nodes, nodeName)
 	// cchain eth api uses a websocket connection and must be closed before stopping the node,
 	// to avoid errors logs at client
-	node.client.CChainEthAPI().Close()
-	if err := node.process.Stop(); err != nil {
-		return fmt.Errorf("error sending SIGTERM to node %s: %w", nodeName, err)
-	}
+	node.GetAPIClient().CChainEthAPI().Close()
+	// Ctrl+C on terminal causes a kill for all process group
+	// so sometimes node is already stopped here
```

Comment:
I don't this sending a SIGTERM to the ANR sends a SIGTERM to the children (i.e. avalanchego). That's why we have the signal handling logic

Reply:
I'm not finding good documentation on that, but Bash at least by default sends SIGINT to the hole process group on Ctrl+C event. That is a very different behaviour than sending SIGINT signal to the process by using `kill -SIGINT pid`
We have signal handling logic to properly release resources, and propagate signals, but that bash case is special.

Reply:
```
package main

import (
    "fmt"
    "os"
    "os/signal"
    "os/exec"
    "syscall"
)

func main() {
    exec.Command("sleep", "100").Start()
	signalChan := make(chan os.Signal, 1)
	signal.Notify(signalChan, syscall.SIGINT)
	sig := <-signalChan
	fmt.Printf("got OS signal %s\n", sig)
}
```
Try to interrupt this with `Ctrl+C` and with `kill -SIGINT` and check the behaviour and if sleep process continue executing.

Reply:
This might be related: https://stackoverflow.com/a/6110213/169252

> The SIGINT signal is generated by the terminal line discipline, and broadcast to all processes in the terminal's _foreground process group_.

Reply:
Yes, the idea you illustrate with your example it is the same idea I wanted to clarify with my example. Ctrl+C kills all processes but `kill -SIGINT pid` only kills parent process.

Reply:
Going back to returning stop errors and improving examples code

Reply:
I think that when a process that uses ANR receives a SIGINT, it shall asume that certain nodes may die before beign signalled to do so. That is sometimes indeed the case. It shall not asume that all the nodes will be still alive.
So, depending how long the user takes to call network Stop(), the signal sent to processes will found them already dead, and this part of the code will return with error.